trunk/SUAVE/Methods/Aerodynamics/Fidelity_Zero/Drag/induced_drag_aircraft.py

```diff
@@ -1,8 +1,8 @@
+
 # induced_drag_aircraft.py
 # 
-# Created:  Dec 2013, SUAVE Team
-# Modified: Jan 2016, E. Botero
-       
+# Created:  Your Name, Dec 2013

```

Comment:
I do not remember changing this file but I can put back what was originally there. This issue has been rectified